CONTRIBUTING.md

```diff
@@ -129,6 +129,20 @@ When you have set up your fork of a repository that you want to contribute to an
 4. **Validating and verifying the PR:** 
    - Reviewers can review the pull request on the GitHub website or fetch the PR using the _Fetch GitHub PR_ menu entry in the _Git Repositories_ view inside the Eclipse IDE.
    - Every PR is automatically verified to check that contributors have a [valid Eclipse Contributor Agreement (ECA)](#setting-up-your-eclipse-and-github-account).
+   ### How does a valid PR look like? (Checklist)
+   These are the basic criteria that need to be fulfilled for a PR to be considered valid. 
+   
+   **If you are a contributor, consider these points as prerequisites before your PR can be reviewed**.
```

Comment:
Add something about inviting people to ask if they don't understand. I don't want people to get "stuck" on something and feel they can't get help just because it can't be reviewed yet.

```suggestion
   **If you are a contributor, consider these points as prerequisites before your PR can be reviewed. Please ask for help if needed, either by making a comment on the PR or the [dev mailing list](#contact).**.
```

Reply:
Done in 465d396b762c1eaf426470b920d807eb1113408b